README.md

```diff
@@ -14,243 +14,228 @@
-| Event | Description|
-| --- | --- |
-|`[instance]_start`|Fires depending on `daily_adjusted_run_time` value and sunrise. You can listen to this event to optimize the moment you irrigate so your irrigation starts just before sunrise and is completed at sunrise. See below for examples on how to use this.|
-|`[instance]_bucketUpd`|Fired when the bucket is calculated. Happens at automatic refresh time or as a result of the `reset_bucket`, `set_bucket` or `calculate_daily_adjusted_run_time` service.|
-|`[instance]_forceModeTog`|Fired when the force mode is disabled or enabled. Result of calling `enable_force_mode` or `disable_force_mode`|
-|`[instance]_hourlyUpd`|Fired when the hourly adjusted run time is calculated. Happens approximately every hour and when `calculate_hourly_adjusted_run_time` service is called.|
+- Name: The name of your zone, e.g. garden
+- Size: The size of this zone (m<sup>2</sup> or sq ft)
+- Throughput: The flow of this zone (liter/minute or gallon/minute)
+- Bucket: You can manipulate this calculated value.
+- Lead Time: In seconds. Time needed to warm up your irrigation system, e.g. time to establish a connection or start a pump etc.
 
-#### Entities
-#### `sensor.smart_irrigation_base_schedule_index`
-The number of seconds the irrigation system needs to run assuming maximum evapotranspiration and no rain / snow. This value and the attributes are static for your configuration.
-Attributes:
-| Attribute | Description |
-| --- | --- |
-|`number of sprinklers`|number of sprinklers in the system|
-|`flow`|amount of water that flows through a single sprinkler in liters or gallon per minute|
-|`throughput`|total amount of water that flows through the irrigation system in liters or gallon per minute.|
-|`reference evapotranspiration`|the reference evapotranspiration values provided by the user - one for each month.|`peak evapotranspiration`|the highest value in `reference evapotranspiration`|
-|`area`|the total area the irrigation system reaches in m<sup>2</sup> or sq ft.|
-|`precipitation rate`|the output of the irrigation system across the whole area in mm or inch per hour|
-|`base schedule index minutes`|the value of the entity in minutes instead of seconds|
-|`auto refresh`|indicates if automatic refresh is enabled.|
-|`auto refresh time`|time that automatic refresh will happen, if enabled.|
-|`force mode duration`|duration of irrigation in force mode (seconds).|
+You can update and calculate all automatic zones.
 
-Sample screenshot:
+**Per Zone Settings**
+You can change any value mentioned before. Additionaly there are some more options.
 
-![](images/bsi_entity.png?raw=true)
+- State: 
+  - 'Automatic': Automatic updating and calculation of that zone. Module and sensor mapping is mandatory.
+  - 'Manual': Only manual updating and calculation of that zone. Module and sensor mapping is mandatory.
```

Comment:
no module and sensor mapping is required in manual mode.

Reply:
done